contentcuration/contentcuration/frontend/channelEdit/components/edit/CompletionOptions.vue

```diff
@@ -778,6 +790,14 @@
       durationRules() {
         const defaultStateForDocument = this.currentCompletionDropdown === null;
         if (this.value) {
+          // duration never required for exercises
+          if (
+            this.value.model === CompletionCriteriaModels.MASTERY ||
+            this.value.model === CompletionCriteriaModels.DETERMINED_BY_RESOURCE ||
+            this.kind === ContentKindsNames.H5P
```

Comment:
Above comment applies here also, maybe?

I also feel like we shouldn't be requiring duration unless they have selected 'duration based' completion criterion in general?

Reply:
same as above